package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "fetch",
-  "version": "0.4.0",
   "main": "fetch.js",
   "private": true,
   "devDependencies": {
     "bower": "1.3.8",
     "jshint": "2.5.2",
-    "node-qunit-phantomjs": "0.2.2"
+    "node-qunit-phantomjs": "0.2.2",
+    "npm-prepublish": "1.0.1"
```

Comment:
Don't need at all — funny, this is the thing that bothers me most about npm :smile:.  Will simplify massively.